mautic/mautic
# [UX] Marketer-friendly labels for dashboard widgets size instead of pixels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Humanized labels help users to get what they want easier.

Discussed on:
https://mautic.slack.com/archives/C024XBCQS3Z/p1711539574454129?thread_ts=1711538952.627039&cid=C024XBCQS3Z



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Dashboard > Add new widget > Height > see new labels

Before:
![image](https://github.com/mautic/mautic/assets/116097999/5e8d2bb4-893a-4e7f-aefa-336aeb2525f5)

After:
![image](https://github.com/mautic/mautic/assets/116097999/77343237-0cf9-4c63-8e13-ccb32b798343)
